cli/command/container/run.go

```diff
@@ -262,7 +262,7 @@ func reportError(stderr io.Writer, name string, str string, withHelp bool) {
 	if withHelp {
 		str += ".\nSee '" + os.Args[0] + " " + name + " --help'"
 	}
-	fmt.Fprintf(stderr, "%s: %s.\n", os.Args[0], str)
+	fmt.Fprintf(stderr, "%s\n", str)
```

Comment:
I am wondering if `fmt.Fprintln(str)` will stream data to stdout, while maybe here we need stderr.